third_party/islet/include/islet.h

```diff
@@ -0,0 +1,83 @@
+/*
+ *  Copyright (c) 2023 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the language governing permissions and
+ *  limitations under the License
+ */
+
+#pragma once
+
+#include <cstdarg>
+#include <cstdint>
+#include <cstdlib>
+#include <ostream>
+#include <new>
+
+enum islet_status_t {
+  ISLET_SUCCESS = 0,
+  ISLET_FAILURE = -1,
+  ISLET_ERROR_INPUT = -2,
+  ISLET_ERROR_WRONG_REPORT = -3,
+  ISLET_ERROR_WRONG_CLAIMS = -4,
+  ISLET_ERROR_FEATURE_NOT_SUPPORTED = -5,
+};
+
+extern "C" {
+
+/// Get an attestation report(token).
+///
+/// # Note
+/// This API currently returns hard-coded report to simulate attest operation.
+/// In future, this will be finalized to support reports signed by RMM.
+/// `User data` could be used as nonce to prevent reply attack.
+islet_status_t islet_attest(const unsigned char *user_data,
+                            int user_data_len,
+                            unsigned char *report_out,
+                            int *report_out_len);
```

Comment:
Thank you for the nice indentation of fn-arguments. It makes the `.h` very much more readable.